src/common/labelValuePair/LabelValuePair.tsx

```diff
@@ -5,21 +5,41 @@ import styles from './labelValuePair.module.scss';
 
 interface Props {
   label: string;
-  value?: string | null;
+  value?: string | null | string[];
   labelColor?: 'standard' | 'brand' | 'critical' | 'secondary';
+  align?: 'left' | 'center' | 'right';
 }
 
 const LabelValuePair = ({
   label,
-  value = '-',
+  value,
   labelColor = 'standard',
-}: Props) => (
-  <div className={styles.labelValuePair}>
-    <span className={classNames(styles.label, styles[labelColor])}>
-      {label}:
-    </span>
-    <span className={styles.value}>{value}</span>
-  </div>
-);
+  align = 'left',
+}: Props) => {
+  let valueArr: React.ReactElement[] = [];
+
+  if (Array.isArray(value)) {
```

Comment:
I don't think this component should be extended to accept array values. The Paragraph component was created for this purpose.